ast/src/input/program_input/program_input.rs

```diff
@@ -58,6 +58,7 @@ impl ProgramInput {
     /// Parse each input included in a file and store them in `self`.
     pub fn parse(&mut self, section: Section) -> Result<(), InputParserError> {
         match section.header {
+            Header::Constants(_constants) => self.main.parse_constants(section.definitions),
```

Comment:
Would it make more sense to store constants in this file since we parse them at this stage?
This would resolve our previous discussion of having to parse both the inputs and constants when calling `MainInput::get`. 

We should know from both program syntax and program input syntax if a variable is a constant -- so the creating additional methods is fine I think.